### ISSUE TITLE
btc steroids

### ISSUE THREAD

**Issue by adulau** [MISP/misp-modules#254]
~~~~
Traceback (most recent call last):
  File "/var/www/MISPcsirt-tooling/venv/lib/python3.5/site-packages/misp_modules/__init__.py", line 198, in post
    response = yield tornado.gen.with_timeout(timeout, self.run_request(jsonpayload))
  File "/var/www/MISPcsirt-tooling/venv/lib/python3.5/site-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
  File "/usr/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/var/www/MISPcsirt-tooling/venv/lib/python3.5/site-packages/misp_modules/__init__.py", line 186, in run_request
    response = mhandlers[x['module']].handler(q=jsonpayload)
  File "modules/expansion/btc_steroids.py", line 142, in handler
    u,e = convert(value, transactions['time'])
  File "modules/expansion/btc_steroids.py", line 49, in convert
    minute, hour = get_consumption()
  File "modules/expansion/btc_steroids.py", line 29, in get_consumption
    minute = str(jreq['Minute']['CallsLeft']['Histo'])
KeyError: 'Minute'
ERROR:misp-modules:Something went wrong:
Traceback (most recent call last):
  File "/var/www/MISPcsirt-tooling/venv/lib/python3.5/site-packages/misp_modules/__init__.py", line 198, in post
    response = yield tornado.gen.with_timeout(timeout, self.run_request(jsonpayload))
  File "/var/www/MISPcsirt-tooling/venv/lib/python3.5/site-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
  File "/usr/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/var/www/MISPcsirt-tooling/venv/lib/python3.5/site-packages/misp_modules/__init__.py", line 186, in run_request
    response = mhandlers[x['module']].handler(q=jsonpayload)
  File "modules/expansion/btc_steroids.py", line 142, in handler
    u,e = convert(value, transactions['time'])
  File "modules/expansion/btc_steroids.py", line 49, in convert
    minute, hour = get_consumption()
  File "modules/expansion/btc_steroids.py", line 29, in get_consumption
    minute = str(jreq['Minute']['CallsLeft']['Histo'])
KeyError: 'Minute'
~~~~

**Comment by rommelfs** [comment]
fixed in 8285ff324fcd03c97ee6a077b05cd5a4430e9712 
The API at https://min-api.cryptocompare.com/stats/rate/limit suddenly changed.

**Comment by adulau** [comment]
Thanks a lot!
